pvnet_app/model_configs/pydantic_models.py

```diff
@@ -0,0 +1,68 @@
+""" A pydantic model for the ML models"""
+
+from typing import List, Optional
+
+import fsspec
+from pyaml_env import parse_config
+from pydantic import BaseModel, Field
+
+
+class ModelHF(BaseModel):
+    repo: str = Field(..., title="Repo name", description="The HF Repo")
+    version: str = Field(..., title="Repo version", description="The HF version")
+
+
+class Model(BaseModel):
+    """One ML Model"""
+
+    name: str = Field(..., title="Model Name", description="The name of the model")
+    pvnet: ModelHF = Field(
+        ..., title="PVNet", description="The PVNet model"
+    )
+    summation: ModelHF = Field(
+        ..., title="Summation", description="The Summation model"
+    )
+
+    use_adjuster: bool = Field(
+        False, title="Use Adjuster", description="Whether to use the adjuster model"
+    )
+    save_gsp_sum: bool = Field(
+        False, title="Save GSP Sum", description="Whether to save the GSP sum"
+    )
+    verbose: bool = Field(
+        False, title="Verbose", description="Whether to print verbose output"
+    )
+    save_gsp_to_forecast_value_last_seven_days: bool = Field(
```

Comment:
Since we have a description of pydantic model, can we shorten the same. Maybe to `save_gsp_to_recent` or something? It is very long!

Reply:
Thats a good idea